api/v1alpha1/nnf_datamovement_types.go

```diff
@@ -87,6 +87,16 @@ type NnfDataMovementConfig struct {
 
 	// Extra options to pass to the dcp command (used to perform data movement).
 	DCPOptions string `json:"dcpOptions,omitempty"`
+
+	// If true, enable the command's stdout to be saved in the log when the command completes
+	// successfully. On failure, the output is always logged.
+	// Note: Enabling this option may degrade performance.
+	LogStdout bool `json:"logStdout,omitempty"`
```

Comment:
Add `+kubebuilder:default:=false` to each of these.

Reply:
Done